hail/src/main/scala/is/hail/expr/types/virtual/TLocus.scala

```diff
@@ -12,21 +12,19 @@ import scala.reflect.{ClassTag, classTag}
 object TLocus {
   def apply(rg: ReferenceGenome): TLocus = TLocus(rg.broadcastRG)
 
-  def apply(rg: ReferenceGenome, required: Boolean): TLocus = TLocus(rg.broadcastRG, required)
-
-  def representation(required: Boolean = false): TStruct = {
-    TStruct(required,
-      "contig" -> +TString(),
-      "position" -> +TInt32())
+  val representation: TStruct = {
+    TStruct(
+      "contig" -> TString,
```

Comment:
()?

Reply:
we want these all to be no-parens eventually, but changing that all here seems like it would add too much noise to the diff in case there were important changes to be caught in review.

Reply:
sure